tests/CSSList/AtRuleBlockListTest.php

```diff
@@ -83,4 +106,19 @@ public function parsesArgumentsOfMediaQueries($css)
 
         self::assertSame('(min-width: 768px)', $atRuleBlockList->atRuleArgs());
     }
+
+    /**
+     * @test
+     *
+     * @param string $css
```

Comment:
We can drop the PHPDoc type annotations for cases where we now can use native type declarations instead.